### DISCUSSION TITLE
Add option for multiple labels for shop packs

### DISCUSSION THREAD

**Discussion by ShadowKurisu** [gameserverapp/Platform#910] [Suggestions]
I can only base this on my perspective in relation to Ark servers. Using maps as an example, there are certain creatures that are only available on certain maps. In addition to restricting which servers the creatures could be delivered to, it would be handy to be able to assign multiple labels (i.e. Server 1, Server 2 etc) so people searching for shop packs that are available for the server they're playing can filter to those labels.

**Comment by mvaessen** [comment]
duplicate of #653
